karoo-gp.py

```diff
@@ -483,8 +483,10 @@ def fx_karoo_pause(model):
         return 2
 
     elif input_a == 'eval':  # evaluate a Tree against the TEST data
-        # Display tree info
-        tree = model.population.next_gen_trees[input_b - 1]
+        if menu_dict['next_gen_len'] > 0:
+            tree = model.population.next_gen_trees[input_b - 1]
+        else:
+            tree = model.population.trees[input_b - 1]
```

Comment:
Should there be any indication that is using the current vs next gen population?

Reply:
That would be good - for now I'm just mimicking the old behavior. There's a lot of duplicated code as well, checking whether to use the `trees` or `next_gen_trees`. I will add a comment for now and lets discuss.
```suggestion
            # TODO: Tell user which population is being used
            tree = model.population.trees[input_b - 1]
```